test/README

```diff
@@ -219,3 +219,32 @@ Still, within a long test, try to have independent sections, where
 each section returns the machine to more or less the state that it was
 in before the section.  This makes it easier to run these sections
 ad-hoc when doing incremental development.
+
+## Running Selenium tests locally
+
+Selenium tests run as a part of GiHub PR CI.
+To run selenium tests locally you will need to do steps described below.
+
+Make sure, first of all, to run:
+
+./bots/image-prepare
+
+By default you'll get corresponding image for system you're running currently.
+E.g. if you run Fedora-27, you'll get Fedora-27 image.
+
+then run:
+
+./bots/image-prepare selenium
+
+you can check which images you currently have:
+
+ls ./bots/images/ | grep .qcow2
+
+Output could be something like this:
+
+fedora-27-b5c5a5bffd5b122d3320d5a8f9743d02f9a61b077f15c9b1917c17f89d1ace9b.qcow2
+selenium-605e2ac857247313a8cd04738baa2f0292545efd3199073cf86a8d9a6083cbef.qcow2
+
+In this case you can run:
+
+./bots/image-prepare fedora-26 && TEST_OS=fedora-27 test/avocado/run-tests --selenium-tests --browser chrome
```

Comment:
It needs to be `TEST_OS=fedora-26`, corresponding to the image-prepare. I'd split this to avoid the overly long line. Also, one might want to run the tests several times without re-preparing the image.